config.go

```diff
@@ -30,3 +30,5 @@ func getEnvOrDefault(key string, defaultValue string) string {
 	}
 	return defaultValue
 }
+
+const GO_SPICE_VERSION = "V6.0"
```

Comment:
Can't we get this from the module?

Reply:
I did look at that, but I don't believe so.